packit_service/models.py

```diff
@@ -278,14 +282,14 @@ def get_or_create(
             if not project_release:
                 project_release = ProjectReleaseModel()
                 project_release.tag_name = tag_name
-                project_release.project_id = project.id
+                project_release.project = project
```

Comment:
interesting, so SA will populate project_id on its own?

Reply:
Yes, it should.